src/items.h

```diff
@@ -312,6 +322,8 @@ class ItemType
 		std::unique_ptr<Abilities> abilities;
 		std::unique_ptr<ConditionDamage> conditionDamage;
 
+		std::map<std::string, bool> hasEdited;
```

Comment:
Here I will use a `unordered_map`, I am hoping to find more things to improve the code